components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -131,6 +136,80 @@ void BravePrefProvider::ShutdownOnUIThread() {
   PrefProvider::ShutdownOnUIThread();
 }
 
+// static
+void BravePrefProvider::RegisterProfilePrefs(
+    user_prefs::PrefRegistrySyncable* registry) {
+  PrefProvider::RegisterProfilePrefs(registry);
+  // Register shields settings migration pref.
+  registry->RegisterIntegerPref(kBraveShieldsSettingsVersion, 1);
+}
+
+void BravePrefProvider::MigrateShieldsSettings(bool incognito) {
+  // Incognito inherits from regular profile, so nothing to do.
```

Comment:
to be precise, "off the record" is true for both Incognito and Guest windows. Incognito window inherits contents settings, while guest session does not.

Reply:
Added a comment about Guest in bdc2851b5386b1ae42a134550a217edb6267ef57